src/sbatch/opt.c

```diff
@@ -2271,13 +2271,14 @@ static void _set_pbs_options(int argc, char **argv)
 {
 	int opt_char, option_index = 0;
 	char *sep = "";
-	char *pbs_opt_string = "+a:A:c:C:e:hIj:J:k:l:m:M:N:o:p:q:r:S:t:u:v:VW:z";
+	char *pbs_opt_string = "+a:A:c:C:d:e:hIj:J:k:l:m:M:N:o:p:q:r:S:t:u:v:VW:z";
 
 	struct option pbs_long_options[] = {
 		{"start_time", required_argument, 0, 'a'},
 		{"account", required_argument, 0, 'A'},
 		{"checkpoint", required_argument, 0, 'c'},
 		{"working_dir", required_argument, 0, 'C'},
+		{"working_dir", required_argument, 0, 'd'},
```

Comment:
why is the `working_dir` the same as previous option?

Reply:
Well, for one, it's not used at all, as far as I can tell (see the `case 'C': break;` below).

Also, `qsub -C` is something else, as far I can tell. From the man page:

```
       -C directive_prefix
               Defines the prefix that declares a directive to the qsub command within the script file.
               See the paragraph on script directives in the Extended Description section.

               If the -C option is presented with a directive_prefix argument that is the null string,
               qsub will not scan the script file for directives.
```

I think that's for changing the `#PBS` that's picked up in job scripts with something else, not related to "working directory" at all.